src/rules/compat.ts

```diff
@@ -156,13 +157,29 @@ export default {
       context.settings?.targets ||
       context.options[0];
 
+    if (
+      !context.settings?.browserslistOpts &&
+      // @ts-expect-error Checking for accidental misspellings
+      context.settings.browsersListOpts
```

Comment:
I can remove this check if you would like me to; however, I have messed up the spelling several times myself, and I would feel so frustrated if I spent a lot of time wondering why it's not working, only to realize I had done `browsersListOpts` instead of `browserslistOpts`

Reply:
this check and the corresponding warning are a great improvement on the ux